frontend/pages/index.tsx

```diff
@@ -0,0 +1,17 @@
+import Head from "next/head";
+
+const Home = () => {
+  return (
+    <div>
+      <Head>
+        <title>WWWeights | Development</title>
+        <meta charSet="utf-8" />
+        <meta name="viewport" content="initial-scale=1.0, width=device-width" />
+      </Head>
+
+      <h1 className="text-blue-500">WWWeights</h1>
+    </div>
+  )
+}
+
+export default Home
```

Comment:
Wir nutzen named exports. Bitte überall named exports

Reply:
"Das macht man so" ist keine gute Antwort. 

Es gibt einige Vorteile von Named imports daher würde ich das auch machen :) Gerne googlen.

Bitte anpassen.

Reply:
Source: https://stackoverflow.com/questions/74281609/is-it-possible-to-use-only-named-exports-in-a-nextjs-project#comment131142592_74281609

> Sure... it's true that [the docs](https://nextjs.org/docs/basic-features/pages) don't explicitly say this... but let me ask you something. How else would the framework know what to import? I mean, if you export some page called MySuperCoolPage, how does Next.js know to import that specifically? Default exports don't have this problem, since they don't have a name.

Im Router Level geht leider kein Named Export da der Router über den default import den Component erkennt. 

@world-wide-weights/frontend
Ich würde aber vorschlagen trotzdem im Code Named Exports zu nutzen (Vorteile: Explicit over implicit, better UX while developing, easier refactoring....) und nur die Router Components werden über default exportiert.

Reply:
#14 Problem ist verschoben in die Diskussionen.